.github/workflows/test-package.yml

```diff
@@ -0,0 +1,34 @@
+name: test-sdk
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    
+    - name: Chekout code
+      uses: actions/checkout@v2
+
+    - name: Setup Dart
+      uses: dart-lang/setup-dart@v1
+    # Setup flutter 
+    - name: "Setup Flutter"
+      uses: subosito/flutter-action@v1
+      with:
+        channel: 'stable' 
+    - name: Install dependencies
+      run: flutter pub get
+
+    #- name: Check for formatting issues (run "flutter format . ")
+    #  run: dart format --set-exit-if-changed .
+
+    - name: Analyze project
+      run: dart analyze
+
+
+    - name: Run tests 1
+      run: dart pub add --dev test
```

Comment:
This shouldn't be necessary: the test dev dependency should be explicitly in the `pubspec.yaml`.